app/controllers/passengers_controller.rb

```diff
@@ -0,0 +1,58 @@
+class PassengersController < ApplicationController
+
+  def index
+      @passengers = Passenger.all.order(:name).page params[:page]
+  end
+
+  def new
+    if params[:passenger_id]
+      passenger = Passenger.find_by(id: passenger_id)
+      @trips = passenger.trips.new
```

Comment:
Hmmmm... I'm curious why the new method in the passengers controller would need to instantiate a trip object.